provider/cmd/cf2pulumi/schema-full.json

```diff
@@ -149549,14 +149545,7 @@
                     "description": "If you are creating a metric stream in a monitoring account, specify true to include metrics from source accounts that are linked to this monitoring account, in the metric stream. The default is false."
                 },
                 "lastUpdateDate": {
-                    "oneOf": [
-                        {
-                            "$ref": "pulumi.json#/Any"
-                        },
-                        {
-                            "$ref": "pulumi.json#/Any"
-                        }
-                    ],
+                    "type": "string",
```

Comment:
This is a breaking change, even if its a very positive one. Again, I'll leave to @danielrbradley for a final call.